core/lib/Cardano/Address/Style/Icarus.hs

```diff
@@ -216,10 +236,9 @@ instance Internal.HardDerivation Icarus where
         in
             Icarus acctXPrv
 
-    deriveAddressPrivateKey (Icarus accXPrv) accountingStyle addrIx =
+    deriveAddressPrivateKey (Icarus accXPrv) role addrIx =
         let
-            changeCode =
-                toEnum @(Index 'Soft _) $ fromEnum accountingStyle
+            changeCode = toEnum @(Index 'Soft _) $ fromEnum role
```

Comment:
For consistency --> `changeCode` --> `roleCode`

Reply:
done in https://github.com/input-output-hk/cardano-addresses/pull/75/commits/e601936c1196486dcee8723fe7be9a20224ce6c7